### PR TITLE
Add support for Wasm plugin

### PR DESCRIPTION
**Pull request by pehala**
* My attempt at updating the extension to support being included in WasmPlugin
* Most likely this means that it will stop working with SME (which is now deprecated anyway)

### CODE REVIEW

**Comment by pehala** [comment]
This should work on boht SME and WASMPlugin now, but it is a bigger change because WasmPlugin requires only one layer so we can have only one COPY command, so I separated the original directory and the crd deployment examples

File: container/manifest.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 schemaVersion: 1
-  
+
```
Comment (eguzki) [inline container/manifest.yaml]: I wonder why this empty line is added

File: container/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM scratch
+
+LABEL type=application/vnd.module.wasm.content.layer.v1+wasm
```
Comment (eguzki) [inline container/Dockerfile:3]: This dockerfile feels wrong. 

Usually it should be two stages, one building the wasm binary and the other only bearing the binary with `scratch` base image.

File: container/manifest.yaml (hunk 1 above)
Comment (pehala) [reply container/manifest.yaml]: My mistake, I will remove it

File: container/Dockerfile (hunk 2 above)
Comment (pehala) [reply container/Dockerfile:3]: I agree. However it was like this before so I didn't change the behaviour. The current process is to do `make release-with-container` (or `make release`) which will build the binary and add it to this directory where you can build the actual image.

**Review by eguzki — APPROVED** [review]

**Review by eguzki — APPROVED** [review]

**Pull request merged**